node/mock/messengerStub.go

```diff
@@ -184,3 +185,12 @@ func (ms *MessengerStub) GetConnectedPeersInfo() *p2p.ConnectedPeersInfo {
 func (ms *MessengerStub) UnjoinAllTopics() error {
 	return nil
 }
+
+// Port -
+func (ms *MessengerStub) Port() int {
```

Comment:
could move the stub to testscommon

Reply:
done, removed duplicated stubs, improved the only common messenger stub